plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/components/header/index.tsx

```diff
@@ -24,7 +25,7 @@ export default function WooPaymentsStepHeader( {
 		<div className="settings-payments-onboarding-modal__header">
 			<img
 				src={ `${ WC_ASSET_URL }images/woo-logo.svg` }
-				alt="Woo Logo"
+				alt={ __( 'Woo Logo', 'woocommerce' ) }
```

Comment:
Best to use "WooCommerce" instead of "Woo" since "Woo" is the business name and also use a placeholder for it (i.e. "%s logo").